stable/keel/templates/deployment.yaml

```diff
@@ -17,6 +17,8 @@ spec:
         app: {{ template "keel.name" . }}
         release: {{ .Release.Name }}
     spec:
+      serviceAccountName: keel
+      serviceAccount: keel
```

Comment:
Don't hard-code the service account. See https://github.com/kubernetes/helm/blob/master/docs/chart_best_practices/rbac.md